templates/repo/issue/view_content/comments.tmpl

```diff
@@ -277,10 +277,12 @@
 			<span class="text grey">
 				{{$.i18n.Tr "repo.issues.dependency.added_dependency" .Poster.HomeLink (.Poster.GetDisplayName|Escape) $createdStr | Safe}}
 			</span>
-			<div class="detail">
-				<span class="octicon octicon-plus"></span>
-				<span class="text grey"><a href="{{$.RepoLink}}/issues/{{.DependentIssue.Index}}">#{{.DependentIssue.Index}} {{.DependentIssue.Title}}</a></span>
-			</div>
+			{{if .DependentIssue}}
+				<div class="detail">
+					<span class="octicon octicon-plus"></span>
+					<span class="text grey"><a href="{{.DependentIssue.HTMLURL}}">{{.DependentIssue.Repo.FullName}}#{{.DependentIssue.Index}} - {{.DependentIssue.Title}}</a></span>
```

Comment:
If current repo != dependentissue.repo ...

Reply:
(Only show repo name if needed)

Reply:
thanks @6543 and @lunny I changed as suggested:

![image](https://user-images.githubusercontent.com/32200834/72775669-e471f300-3bd4-11ea-9ff6-3aaa170582cc.png)